source/opentelemetry-collector-contrib/examples/tracing/main.go

```diff
@@ -0,0 +1,166 @@
+// Copyright -c Google LLC
```

Comment:
since you are adding go and python examples so it make sense to refactor this into two separate directories
under examples tracing 1) go 2) python and move the corresponding files to corresponding directory.